src/sentry/digests/backends/redis.py

```diff
@@ -218,9 +239,6 @@ def add(self, key, record):
                 ex=self.ttl,
             )
 
-            pipeline.set(make_iteration_key(timeline_key), 0, nx=True)
-            pipeline.expire(make_iteration_key(timeline_key), self.ttl)
```

Comment:
This previously set value will just TTL out, it shouldn't have been set frequently enough to be of any significance.